index.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const fsCallback = require("./lib/files-callBack.js");
+const fsPromise = require("./lib/file-promise.js"), write = require("./lib/write");
```

Comment:
`fsPromise` and `write` should be defined on separate lines

Reply:
Because of this error, your code is failing and cannot be run: 
<img width="949" alt="Screen Shot 2019-10-27 at 9 37 56 PM" src="https://user-images.githubusercontent.com/22336391/67652431-193a1b00-f902-11e9-9c95-cb82f4cf4660.png">

Reply:
I did it properly , to be honest it was like this before I am not sure how two line of requirements are on same line